### PR TITLE
Junit testing 

### PR DESCRIPTION
**Pull request by CarrieLi1015**
Testing added:
1. filter command test
2. filter command parser test 
3. addScore command test
4. addScore command parser test
5. switch command test
6. deleteScore command test
7. TagContainsKeywordsPredicate test
8. markTaskComplete parser test
9. markTaskInProgress parser test
10. markTaskLate parser test

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/DeleteScoreCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import seedu.address.model.UserPrefs;
+
+class DeleteScoreCommandTest {
+
+    private Model model = new ModelManager(getTypicalMathutoring(), new UserPrefs());
+
+    /*@Test
+    public void constructor_nullIndexes_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new DeleteScoreCommand(null, null));
+    }*/
```
Comment (toh-xinyi) [inline src/test/java/seedu/address/logic/commands/DeleteScoreCommandTest.java:26]: Can remove this part if it is not used

**Review by toh-xinyi — COMMENTED** [review]
LGTM!

File: src/test/java/seedu/address/logic/commands/DeleteScoreCommandTest.java (hunk 1 above)
Comment (CarrieLi1015) [reply src/test/java/seedu/address/logic/commands/DeleteScoreCommandTest.java:26]: To keep this part or not is depends on whether Qianhui wanna change part of the java code. The test case is valid, but fail cause on line missing in deleteScoreCommand.

Comment (toh-xinyi) [reply src/test/java/seedu/address/logic/commands/DeleteScoreCommandTest.java:26]: Ohh I see, I see. Okay!

Comment (toh-xinyi) [reply src/test/java/seedu/address/logic/commands/DeleteScoreCommandTest.java:30]: @QQH0828 will you be changing part of the java code to allow for the addition of this test case?

Comment (QQH0828) [reply src/test/java/seedu/address/logic/commands/DeleteScoreCommandTest.java:30]: Yes, I will add this Java code after confirming with Yu Ting.

**Pull request merged**